test/controllers/ConfirmationController.test.ts

```diff
@@ -22,11 +22,12 @@ describe('ConfirmationController', () => {
 
         const appeal = {
             penaltyIdentifier: {
-                companyNumber: '00345567',
+                companyNumber: 'NI000000',
             },
         } as Appeal;
 
         const applicationData = {
+            appeal,
```

Comment:
I see you included `appeal` property here. Will `appeal` still be present after submission? Isn't it moved to `submittedAppeal`? I am worrying that if `appeal` is not present it will break confirmation page.

Reply:
you're right, the `appeal` is not available after submission, but the test is failing due to the CompanyAuthMiddleware expecting a company number on the `appeal` object. Thinking about it, the CompanyAuthMiddleware doesn't need to apply to this page since the appeal has already been submitted.. i'll clean this up

Reply:
If test is failing and appeal data is not going to be there that live app would misbehave as well. I think company auth middleware might not be needed for confirmation page cause its purpose is to stop people appealing on other people behalf without code and at that point appeal is already done.

Reply:
after removing CompanyAuthMiddleware from the ConfirmationController, test passes. i've checked we're referencing the correct data